backend/routes/addUnknowns.js

```diff
@@ -0,0 +1,96 @@
+const express = require('express');
+const router = express.Router();
+const admin = require('firebase-admin');
+
+// Add Unknowns Endpoint
+router.post('/addUnknowns', async (req, res) => {
+  try {
+    // Extract location and occupied courts from the request body
+    const { location, occupiedCourts } = req.body;
+    if (!location || !Array.isArray(occupiedCourts)) {
+      return res.status(400).json({ message: 'Location and occupied court information is required.' });
+    }
+
+    // Retrieve active court information for the given location
+    const activePlayersSnapshot = await admin.firestore()
+      .collection('locations')
+      .doc(location)
+      .collection('activePlayers')
+      .orderBy('courtNumber')
```

Comment:
Does it need to be ordered in this case? I believe that removing .orderBy() would improve the runtime.